data/framestruct/converter.go

```diff
@@ -152,7 +152,14 @@ func (c *converter) convertMap(toConvert interface{}, tags, prefix string) error
 		return errors.New("map must be map[string]interface{}")
 	}
 
-	for name, value := range m {
+	var names []string
```

Comment:
Maybe makes sense to preallocate capacity here?

Reply:
That's reasonable. It's done in the latest commit